package.json

```diff
@@ -66,9 +66,9 @@
     "ember-cli-babel": "^5.1.7",
     "ember-cli-htmlbars": "^1.1.1",
     "ember-cli-sass": "5.6.0",
-    "ember-frost-core": "3.0.0"
+    "ember-frost-core": "^3.0.0"
```

Comment:
This is not related to the context of this PR.  Do not mix efforts.  Remove this change.